apply.sh

```diff
@@ -0,0 +1,23 @@
+repos=(
+  alphagov/design-system-team-internal
+)
+
+args=(
+  --allow-added-labels # Keep any existing labels that are not in the config
```

Comment:
Will this mean if we change the labels that old ones will be left behind, requiring a manual cleanup?

Reply:
Yep. We can consider removing this in the future, once we're happy we've got all the labels we need everywhere? The PR would show us the changes.

For now, this is documented in the README.